packages/react-native-bridge/android/src/main/java/org/wordpress/mobile/ReactNativeGutenbergBridge/RNReactNativeGutenbergBridgeModule.java

```diff
@@ -228,8 +228,8 @@ public void requestMediaEditor(String mediaUrl, final Callback onUploadMediaSele
     }
 
     @ReactMethod
-    public void requestStoryCreatorLoad(ReadableArray mediaFiles, String blockId) {
-        mGutenbergBridgeJS2Parent.requestStoryCreatorLoad((savedMediaFiles, savedBlockId) ->
+    public void requestMediaFilesEditorLoad(ReadableArray mediaFiles, String blockId) {
```

Comment:
##### Alternative to use `sendActionButtonPressed`
It was discussed previously that we could probably use `sendActionButtonPressed` in place of the dedicated `requestMediaFilesEditorLoad` (previously named `requestStoryCreatorLoad`) but I understand the `sendActionButtonPressed` exists to signal the host app about something, and then changing state and then being able to actually trigger the fallback (as for example `requestUnsupportedBlockFallback ` in the following piece of code), and I believe this is because we had a bottom sheet appearing for the unsupported block case.

```
				onModalHide={ () => {
					if ( this.state.sendFallbackMessage ) {
						// On iOS, onModalHide is called when the controller is still part of the hierarchy.
						// A small delay will ensure that the controller has already been removed.
						this.timeout = setTimeout( () => {
							// for the Classic block, the content is kept in the `content` attribute
							const content =
								blockName === 'core/freeform'
									? attributes.content
									: attributes.originalContent;
							requestUnsupportedBlockFallback(
								content,
								clientId,
								blockName,
								blockTitle
							);
						}, 100 );
						this.setState( { sendFallbackMessage: false } );
					} else if ( this.state.sendButtonPressMessage ) {
						this.timeout = setTimeout( () => {
							sendActionButtonPressedAction(
								actionButtons.missingBlockAlertActionButton
							);
						}, 100 );
						this.setState( { sendButtonPressMessage: false } );
					}
				} }

```

Given we don't have a bottom sheet but rather have a direct signal to trigger the media files editor (in particular, the story block editor), I think just making the previously named `requestStoryCreatorLoad` bridge method more generic is enough (as it is now).

I have saved my work though in these following branches, in case there's a compelling reason to go with the 2-phased signaling of 1) `sendActionButtonPressedAction`  and then 2) a similarly named to `requestUnsupportedBlockFallback` method, because otherwise we would need to add all these optionals to `sendActionButtonPressedAction`: blockId, mediaFiles, and the `ReplaceMediaFilesEditedBlockCallback` callback function as well. I don't think it's necessary, but interested in knowing what your thoughts on this are @etoledom 🙇 

git push --set-upstream origin try/jetpack-stories-try-send-action

The WIP branches showing this exploration are all the same name on all 4 repos: (WPAndroid, gutenberg-mobile, jetpack, and gutenberg):
base branch: try/jetpack-stories-block-mobile-on-done-refactor
exercise  branch: try/jetpack-stories-try-send-action

Reply:
We could add an optional block ID and callback block but I agree that the need of `mediaFiles` as an argument makes it specific enough to have its own method 👍